### PR TITLE
Add features to Jakartaee9 feature

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature [hunk 1]
```diff
@@ -7,8 +7,17 @@ IBM-ShortName: jakartaee-9.0
...
-  com.ibm.websphere.appserver.transaction-2.0
-kind=noship
-edition=full
+ com.ibm.websphere.appserver.el-4.0, \
+ com.ibm.websphere.appserver.jsp-3.0, \
+ com.ibm.websphere.appserver.servlet-5.0, \
+ com.ibm.websphere.appserver.jsonp-2.0, \
+ com.ibm.websphere.appserver.jsonb-2.0, \
+ com.ibm.websphere.appserver.transaction-2.0, \
+ io.openliberty.jakarta.cdi-3.0, \
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:16]: adding these private api features here should not be required? They should be pulled in transitively by whichever public features depend on them?

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature [hunk 2]
```diff
@@ -7,8 +7,17 @@ IBM-ShortName: jakartaee-9.0
...
+ io.openliberty.jakarta.interceptor-2.0, \
+ com.ibm.websphere.appserver.jdbc-4.0, \
+ com.ibm.websphere.appserver.jdbc-4.1, \
+ com.ibm.websphere.appserver.jdbc-4.2, \
+ com.ibm.websphere.appserver.jdbc-4.3
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:21]: I don't think you can pull in multiple versions of jdbc like this... you need to pull in just 4.3 and then optionally tolerate the others. ee8 pulls in 4.2 and tolerates 4.3 ... this should be the opposite I think?

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
Build failed because of connection timeout...   restarted here: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_PdohQLfEEeqI3q2xeI2nKA

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#libby

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature (hunk 1 above)
Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:17]: Moreover why the change in direction to not update webProfile-9.0 and have jakartaee-9.0 pull in webProfile-9.0?

Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:17]: jakartaee-9.0 isn't core either.

Comment (cbridgha) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature:17]: Hi Completely agree regarding changes to jakartaee feature - This was a temporary solution to release the beta content, and ensure some private API's were made available. - I expect webprofile-9.0 and other tweaks to the features will be ready by next beta - (actually this week)

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature (hunk 2 above)
Comment (cbridgha) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature]: This was fixed...

**Pull request merged**